### PR TITLE
EUI-8608 QM follow up on the query tab

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8608

### Change description ###
Query Management - Follow-up UI on the query tab

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details/query-details.component.spec.ts [hunk 1]
```diff
@@ -1,21 +1,124 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { QueryDetailsComponent } from './query-details.component';
+import { QueryListItem } from '../../models';
+import { By } from '@angular/platform-browser';
+import { MockRpxTranslatePipe } from 'projects/ccd-case-ui-toolkit/src/lib/shared/test/mock-rpx-translate.pipe';
```
Comment (connorpgpmcelroy) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details/query-details.component.spec.ts:6]: Imports incorrectly ordered and reference to entire folder not needed, should be ../lib at most

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Needs import changes

**Review by Daniel-Lam — APPROVED** [review]
@johnbenjamin-hmcts - all looks good here; approved

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details/query-details.component.spec.ts (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details/query-details.component.spec.ts]: @connorpgpmcelroy - I have now updated the code.

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details/query-details.component.spec.ts (hunk 1 above)
Comment (connorpgpmcelroy) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details/query-details.component.spec.ts:4]: still references entire ccd-case-ui-toolkit folder, should be relative link '../../'

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Minor change requested

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by Daniel-Lam — APPROVED** [review]
@johnbenjamin-hmcts - all looks good here; approved

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by Daniel-Lam — APPROVED** [review]
@johnbenjamin-hmcts - re-approved after base branch change

**Pull request merged**